packages/node/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "types": ["jest", "node"],
```

Comment:
As mentioned before, the root `tsconfig.json` has `"types: ["jest"]`, which excludes other types - but we need `@types/node` here hence why it is redefined.

It would be better to remove `"types: ["jest"]` from the root `tsconfig.json` but doing so means that `@types/react-native` are also included, which then conflict with lib dom, as defined by `"lib": [ "dom", "esnext" ],`.

So to do that we'll need to restructure the way jest / typescript is set up. One for a later PR.